plugins/continuous-toolbox/src/ContinuousToolbox.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Toolbox that uses a continuous scrolling flyout.
+ */
+
+import * as Blockly from 'blockly/core';
+
+/**
+ * Class for continuous toolbox.
+ */
+export class ContinuousToolbox extends Blockly.Toolbox {
+  /** @override */
+  constructor(workspace) {
+    super(workspace);
+  }
+
+  /** @override */
+  init() {
+    super.init();
+
+    // Populate the flyout with all blocks and show it immediately.
+    let contents = [];
+    for (const toolboxItem of this.contents_) {
+      if (toolboxItem.isSelectable()) {
+        // Create a label node to go at the top of the category
+        contents.push({kind: 'LABEL', text: toolboxItem.name_});
```

Comment:
I'm curious if we would want to add this to the toolbox contents or not. It doesn't seem to make a difference for what we are currently doing, but I am trying to figure out if it might be useful for an external use case. 

Just a thought, not something to address in this PR.